d2l-table-circle-button.js

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
This was only referenced from scroll-wrapper, so has been deleted.

Reply:
Is the plan just to fix these tests after? https://search.d2l.dev/xref/lms/uiautomation-tests/D2L.Automation.UI.PageObjects/Dropbox/DropboxFoldersListPage.cs?r=58509bd2#274